Tests/UnboxTests.swift

```diff
@@ -1358,6 +1358,27 @@ class UnboxTests: XCTestCase {
         }
     }
     
+    func testUnboxingArrayOfStringTransformedToInt() {
+        let dictionary: UnboxableDictionary = ["intArray": ["123", "456", "789"]]
+        
+        struct ModelA: Unboxable {
+            let intArray: [Int]
+            init(unboxer: Unboxer) {
+                intArray = unboxer.unbox("intArray")
```

Comment:
To be consistent with the rest of the codebase, could you explicitly reference `self`? 😇  (yeah, I'm one of those people 😝)